extension/src/experiments/columns/model.ts

```diff
@@ -45,12 +45,21 @@ export class ColumnsModel extends PathSelectionModel<Column> {
   }
 
   public getFirstThreeColumnOrder(): string[] {
-    return this.columnOrderState
-      .filter(
-        path =>
-          this.status[path] && this.status[path] === 2 && path !== 'Created'
-      )
-      .slice(0, 3)
+    const metrics: string[] = []
+    const params: string[] = []
+
+    for (const path of this.columnOrderState) {
+      if (this.status[path] !== 2) {
+        continue
+      }
+      if (path.startsWith(ColumnType.METRICS)) {
```

Comment:
[C] If there are already 3 we could continue (add `||`s to the initial if statement) or could also add the condition to the if statement (could be helpful with lots of params/metrics)